doc/classes/MainLoop.xml

```diff
@@ -215,5 +215,13 @@
 			Notification received from the OS when an update of the Input Method Engine occurs (e.g. change of IME cursor position or composition string).
 			Specific to the macOS platform.
 		</constant>
+		<constant name="NOTIFICATION_APP_RESUMED" value="1014">
```

Comment:
You forgot to bind those constants in MainLoop, so this documentation gets removed. I'll fix it.